100% coverage

jo opened the pull request:
- [ ] update https://github.com/eHealthAfrica/kazana/issues/131 when merged

jo commented:
Blocked by https://github.com/eHealthAfrica/kazana-server/pull/109

gr2m commented:
https://github.com/eHealthAfrica/kazana-server/releases/tag/v3.9.6 released. I restarted the travis build, now getting `Cannot find module 'boom'` error

jo commented:
Thanks @gr2m :+1: 
This is still WIP, sorry if my last comment was confusing. Will continue tomorrow.

jo commented:
`[boom` was a devDependency](https://github.com/eHealthAfrica/kazana-raw-data/blob/master/package.json#L40) which I have removed when I replaced the mock server with a real one.

This would have caused a disfunctional deploy if we'd used `npm install --production`, which we do AFAIK. It reminds me that we should use the production flag in our integration tests.

jo commented:
WIP: This still needs tests for most attachment api endpoints.

jo commented:
This is ready now

jo commented:
please wait, I'll try to reach 100% branch coverage

jo commented:
done. Branch coverage is now at 100%.

FrancescoK commented:
:clap: :clap: :clap:
:tada:

gr2m commented on lib/routes/submissions/update.js:
```diff
@@ -34,6 +34,8 @@ function updateSubmissionHandler (request, reply) {
 
   var promise = user.isAdmin
+  // FIXME
+    // ? store.find(id)
```
what’s this?

jo replied:
oh, leftover. Will remove that comment and have a look

jo replied:
I have removed that obsolete comment.

gr2m commented:
very nice :+1:

jo commented:
let me see what happens if I try to merge with pending checks...

jo commented:
hmmm it worked.

jo commented:
WTF?

jo commented:
I was able to merge the pr regardless of the status checks:

![screenshot from 2016-01-14 17-26-42](https://cloud.githubusercontent.com/assets/17586/12330247/7218cc7c-bae4-11e5-9857-8ddfb5a40028.png)
